teaser/data/output/modelicatemplate/AixLib/AixLib_ThermalZoneRecord_FourElement

```diff
@@ -15,7 +15,7 @@ record ${zone.parent.name}_${zone.name} "${zone.parent.name}_${zone.name}"
     alphaWin = ${zone.model_attr.alpha_conv_inner_win},
     RWin = ${zone.model_attr.r1_win},
     gWin = ${zone.model_attr.weighted_g_value},
-    UWin= ${zone.model_attr.ua_value_win/zone.model_attr.area_win},
+    UWin= ${zone.model_attr.ua_value_win},
```

Comment:
should this be `zone.model_attr.u_value_win` instead of `zone.model_attr.ua_value_win`